carbonj.service/src/main/java/com/demandware/carbonj/service/engine/cfgCarbonJ.java

```diff
@@ -654,22 +598,17 @@ public ConfigServerUtil configServerUtil(ScheduledExecutorService s, RestTemplat
                 server.setAttribute( "org.eclipse.jetty.server.Request.maxFormContentSize", jettyMaxFormContentSize );
 
                 // TODO change to use logback ch.qos.logback.access.jetty.RequestLogImpl?
-                NCSARequestLog ncsaLog = new NCSARequestLog( jettyLogfilePath );
-                ncsaLog.setExtended( true );
-                ncsaLog.setAppend( true );
-                ncsaLog.setLogTimeZone( "GMT" );
-                ncsaLog.setRetainDays( jettyRequestLogRetentionDays );
+//                NCSARequestLog ncsaLog = new NCSARequestLog( jettyLogfilePath );
```

Comment:
Removing request log?

Reply:
Good catch. The class was removed from Jetty. Not sure how to get this functionality back.